src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -190,8 +197,11 @@ async function startToolSetupFlow(
     case 'tsc':
       await configureSentryCLI(options, configureTscSourcemapGenerationFlow);
       break;
-    default:
+    case 'create-react-app':
       await configureSentryCLI(options);
       break;
+    default:
+      await configureSentryCLI(options, configureCRASourcemapGenerationFlow);
```

Comment:
I might be missing some context - shouldn't this second param go into the `create-react-app` case?

Reply:
Oh you're totally right!